_includes/landing-page/landing-description.html

```diff
@@ -0,0 +1,68 @@
+<div class="container">
+    <div class="columns is-multiline">
+        <div class="column is-12-tablet is-6-desktop is-spaced-bottom">
+            <h1 class="title is-size-3-tablet is-size-2-desktop has-text-grey-darker">
+                Run functions anywhere enabling youwith the same unified experience.
```

Comment:
Missed a space here. Looking at this locally in VSCode, there appears to be some special character, so you may not be able to see it in your dev environment.

![Screenshot from 2019-04-17 19-52-25](https://user-images.githubusercontent.com/12687192/56330185-95ca9080-614c-11e9-8b42-cab9497ceb02.png)